build/patches/Disable-conversion-measurement-api.patch

```diff
@@ -209,14 +202,6 @@ diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5
      {
        name: "AbortSignalThrowIfAborted",
        status: "stable",
-@@ -482,7 +489,7 @@
-     },
-     {
-       name: "ConversionMeasurement",
```

Comment:
Removed in Remove HTML-attribute-based attribution source registration | https://chromium-review.googlesource.com/c/chromium/src/+/3576874